frontend/packages/dev-console/src/components/topology/helm-details/TopologyHelmReleaseNotesPanel.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { SyncMarkdownView } from '@console/internal/components/markdown-view';
+
+type TopologyHelmReleaseNotesPanelProps = {
+  releaseNotes: string;
+};
+
+const TopologyHelmReleaseNotesPanel: React.SFC<TopologyHelmReleaseNotesPanelProps> = ({
+  releaseNotes,
+}) => (
+  <div className="overview__sidebar-pane-body">
+    {releaseNotes && <SyncMarkdownView content={releaseNotes} />}
```

Comment:
Helm specifically says the notes are plain text: https://helm.sh/docs/chart_template_guide/notes_files/

I wonder if we're better off just rendering in a pre-formatted block. There is potential for errors if the plain text matches markdown.

Reply:
I found one example of notes that uses markdown for a snippet: https://github.com/helm/charts/blob/master/stable/wordpress/templates/NOTES.txt
but a bunch of others were plain text.

Reply:
I noticed a couple in markdown as well during development which is why I went with the MD viewer.